src/applications/static-pages/facilities/tests/OtherFacilityListWidget.unit.spec.jsx

```diff
@@ -29,11 +29,6 @@ describe('facilities <OtherFacilityListWidget>', () => {
       tree.update();
       expect(tree.find('va-loading-indicator').exists()).to.be.false;
 
-      const facilityName = tree.find('h3');
```

Comment:
This has to be removed because now its contents are a web component and we can't assert on the shadow DOM in unit tests.

Reply:
yay shadow DOM